examples/basic/mod.ts

```diff
@@ -0,0 +1,16 @@
+import { DOMParser } from "https://deno.land/x/deno_dom@v0.1.2-alpha4/deno-dom-wasm.ts";
+import type { Element } from "https://deno.land/x/deno_dom@v0.1.2-alpha4/deno-dom-wasm.ts";
```

Comment:
Please remove the specific version `@v0.1.2` from the imports in favor of implicit version selection

Reply:
Also, rename this file to `examples/basic.ts`  we don't need a folder here